Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Copyright Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-FROM docker.io/library/golang:1.21.5-alpine3.18@sha256:d8b99943fb0587b79658af03d4d4e8b57769b21dcf08a8401352a9f2a7228754 as builder
+FROM docker.io/library/golang:1.21.6-alpine3.18@sha256:869193e7c30611d635c7bc3d1ed879039b7d24710a03474437d402f06825171e as builder
```

Comment:
We should probably consider updating to Alpine 3.19.

Reply:
Good point. Will open a PR updating it after this PR is merged.

Reply:
https://github.com/cilium/cilium-cli/pull/2225